src/content/structured/components/divider/guidance.mdx

```diff
@@ -0,0 +1,188 @@
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={dividerFig4}
+    imageAlt="Full-width dividers in the 'dark' theme colour."
+    state="good"
+    caption="The 'dark' theme colour on the divider component."
+  />
+  <DoDontCaution
+    imageSrc={dividerFig5}
+    imageAlt="Full-width dividers in the 'light' theme colour."
+    state="good"
+    caption="The 'light' theme colour on the divider component."
+  />
+</DoubleDoDontCaution>
+
+### Weight
+
+The divider component has four weights to choose from. These are thin, medium, thick and very thick.
+
+Alter the weight of the divider to create a clear and logical structure of the page; for example, using a thicker divider to clearly separate unrelated content, whilst using thinner dividers to group together less important content.
+
+<DoDontCaution
+  imageSrc={dividerFig6}
+  imageAlt="Divider in 'light' theme colour and 8px weight used to separate two sections of a page, whilst various monochrome dividers with 1px weight separate less important content."
+  state="good"
+  caption="Use different weights of dividers to show the structure of a page."
+/>
+
+### Label and orientation
+
+Pair dividers with labels to help define content groupings. Place the label within the divider to reinforce the label's connection to the content below the divider.
+
+For the horizontal divider, the alignment of the label can be left-aligned, centred, or right-aligned. For the vertical divider, the label can also be centred, or placed at the top or bottom of the divider.
+
+Make sure the text in the label is brief so that it clearly aids navigation and page structure. If not, the divider component becomes overwhelming. When the text reaches a certain amount, it will go onto multiple lines which disregards the divider's purpose. The information placed here should be non-essential with textual content placed inside the page content itself.
+
+<DoubleDoDontCaution>
+  <DoDontCaution
+    imageSrc={dividerFig7}
+    imageAlt="Divider with left-aligned 'Advanced' label being used to separate sections in Settings."
+    state="good"
+    caption="Divider being placed to the right of the label to reinforce the label's connection to the content."
+  />
+  <DoDontCaution
+    imageSrc={dividerFig8}
+    imageAlt="Divider with left-aligned label that has a word count which is not recommended. Multiple lines are required to show the lengthy label."
+    state="bad"
+    caption="Divider with label text that is too long."
+  />
+</DoubleDoDontCaution>
+
+Use horizontal or vertical dividers to create any necessary sections between content. Pair the vertical divider with text to create a heading when you want to draw attention to a paragraph.
```

Comment:
```suggestion
Use horizontal or vertical dividers to create any necessary sections between content. Pair the vertical divider with text to create a heading when you want to draw attention to a paragraph.
```

The heading is non-semantic so not sure if the word "heading" is misleading. Maybe use the word "title" instead